modules/release-notes/ref-release-notes-compatibility-matrix.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: REFERENCE
+[id="compatibility-matrix"]
+= Compatibility matrix
+
+The following table lists products and their versions compatible with {product-short} {product-version}:
+
+.Platform Compatibility Matrix
+[cols="1,2,1", options="header"]
+|===
+| {product-short} | Platform | {product-custom-resource-type} Version
+
+| `{product-version}`
+|
+- {ocp-brand-name} `{ocp-version-min}`-`{ocp-version}`
+
+- {eks-brand-name} ({eks-short})
+
+- {aks-brand-name} ({aks-short})
+
+- {aro-brand-name} ({aro-short})
```

Comment:
aro and rosa are not yet in the attributes.adoc file:

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/release-1.4/artifacts/attributes.adoc#L73-L87

So you'll want to add those before this will render correctly.

The preview looks like this:

![image](https://github.com/user-attachments/assets/0a8de18d-56ee-40b5-9662-19d6d3c072ba)